nifpga/bitfile.py

```diff
@@ -32,12 +32,15 @@ def __init__(self, filepath, parse_contents=False):
             if reg.datatype is not None:
                 assert reg.name not in self._registers, \
                     "One or more registers have the same name '%s', this is not supported" % reg.name
-                self._registers[reg.name] = reg
+                if not reg.is_array():
+                    self._registers[reg.name] = reg
```

Comment:
Won't this break all arrays?  I think you meant to call _is_register_fxp?

Reply:
Opps fixed. want to check both array and FXP